test/testinfra.py

```diff
@@ -401,22 +423,81 @@ def update_status(revision, context, last, changes):
                     if status.get("description", NO_TESTING) == NO_TESTING:
                         baseline = 0
 
-                (priority, changes) = self.prioritize(status, labels, baseline)
+                (priority, changes) = self.prioritize(status, labels, baseline, context)
                 if update_status(revision, context, status, changes):
-                    results.append((priority, "pull-%d" % number, revision, "pull/%d/head" % number, context))
+                    results.append(GitHub.TaskEntry(priority, GithubPullTask("pull-%d" % number, revision,
+                                                                             "pull/%d/head" % number, context)))
+
+    def scan_for_image_tasks(self):
+        # XXX
+        return [ (10, GithubImageTask("refresh-fedora-23", "fedora-23")) ]
+
+        issues = self.get("issues?labels=bot")
+
+        results = [ ]
+        for image in DEFAULT_IMAGE_REFRESH:
+            found = False
+            for issue in issues:
+                if issue['title'] == ISSUE_TITLE_IMAGE_REFRESH.format(image):
+                    age = time.time() - time.mktime(time.strptime(issue['created_at'], "%Y-%m-%dT%H:%M:%SZ"))
+                    if age < IMAGE_REFRESH * 24 * 60 * 60:
+                        found = True
+                    print "not old enough", image, age
+            if not found:
+                results.append(GitHub.TaskEntry(10, GithubImageTask("refresh-" + image, image)))
```

Comment:
we should make "baseline" a variable, and let the master priority and this one depend on that
https://github.com/cockpit-project/cockpit/blob/0a95300a0abbe0ac6e1497b3ab2c573f70d50ab6/test/testinfra.py#L389